apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,150 @@ const data: FoundAnimalFlow = {
+                      "This is a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: [
+                      "This is an infant squirrel, You will need to guide the baby back to its mother",
+                      "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds, Wrap the sock in a soft towel, and place it with the baby squirrel in an open container (e.g. a box)",
+                      "Do NOT give the baby food or water",
+                      "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found) If you dont know which tree the squirrels nest is in, or if the nest  was destroyed. Then choose a tree closest to where the squirrel was found",
+                      "Observe the baby for the next 6 to 8 hours, Reheat the rice every two hours. If the mother does return after 24 hours, you can...",
+                      ...macros.general.rehab("the squirrel"),
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Cat",
+      flow: {
+        prompt: [
+          "Does the cat appear to be sick, injured, in danger, or a nursing kitten with no mama in sight?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "If the cat appears to be sick, injured, in danger, or if you've found a nursing kitten with no mama in sight please contact an animal welfare organization or shelter that can help you assess the situation",
+              ],
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Many cats that people spot in neighborhoods aren't in need of immediate help. Instead, they are pets who spend some or all of their time outdoors.",
+                "Check for a collar, if the cat has one try and get in contact with the owner, If the cat has no collar you can take the cat to the nearest animal shelter to check for a microchip",
+                "If the cat does not have a microchip, assess whether the cat seems healthy (not injured or visibly ill) and safe (in a secure location, away from any danger) We want a healthy and safe cat to stay where it is while you attempt to find the owner",
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Raccoon",
+      flow: {
+        prompt: [
+          "Does the raccoon appear to be sick or injured?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "Is the raccoon an adult or a baby?",
```

Comment:
Like the other flows, we probably want some guidance here on how to tell apart an adult/baby, even if just a rough size?